server_installation/topics/operator/keycloak-user-cr.adoc

```diff
@@ -0,0 +1,76 @@
+
+=== KeycloakUser Custom Resource
+
+{project_operator} allows application developers to represent {project_name} Users as Custom Resources:
```

Comment:
Why these sentences (also for Keycloak realms and clients above) need to be written in so a complex way?

Why not just `{project_name} Users can be represented as Custom Resources in {project_operator}.`?

Or `Use Custom Resource in {project_operator} to represent {project_name} Users`

Reply:
Fixed.